addons/web_tour/static/src/scss/tip.scss

```diff
@@ -3,15 +3,16 @@
     position: relative!important;
 
     // Tooltips are placed in the <body/> element when opened because this is
-    // the only way to position them above everything else thanks to a 1051
+    // the only way to position them above everything else thanks to a 1052
```

Comment:
the tooltip will have z-index 1070, but this is needed because the highest element in Odoo (except tooltip) has z-index 1052. So I took the sentence as "thanks to a 1052 z-index somewhere in Odoo we have to have a equal z-index (or bigger) for tooltip".

Reply:
well i'm not sure to understand this sentence actually

Reply:
i know but when it says 'thanks to a 1052 z-index', i understand that we put the z-index to 1052 to ensure to be above everything, whereas you seem to understand that it states that the 'higher' element in odoo has z-index 1052.

The sentence is a bit weird, so i don't know. But if i'm right, in the sentence 1052 should be replaced by 1070.